modin/test/test_partition_api.py

```diff
@@ -96,3 +116,53 @@ def test_from_partitions(axis):
             futures = client.scatter([df1, df2], hash=False)
     actual_df = from_partitions(futures, axis)
     df_equals(expected_df, actual_df)
+
+
+@pytest.mark.parametrize("row_indices", [[0, 2], slice(None)])
+@pytest.mark.parametrize("col_indices", [[0, 2], slice(None)])
+@pytest.mark.parametrize("is_length_future", [False, True])
+@pytest.mark.parametrize("is_width_future", [False, True])
+def test_mask_preserve_cache(
+    row_indices, col_indices, is_length_future, is_width_future
+):
+    def serialize(obj):
```

Comment:
naming seems wrong... it's a `deserialize` at least :)

Reply:
right)

fixed it